databases/litecli/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        dbcli litecli 1.0.0 v
+revision            0
+
+categories          databases python
+maintainers
```

Comment:
This needs to be `nomaintainer` if @xeron does not respond or choose to be maintainer.

Reply:
set to nomaintainer